src/ui-player.c

```diff
@@ -1222,31 +1222,37 @@ void do_cmd_change_name(void)
 			switch (ke.key.code) {
 				case ESCAPE: more = false; break;
 				case 'c': {
-					char namebuf[32] = "";
 
-					/* Set player name */
-					if (get_character_name(namebuf, sizeof namebuf))
-						my_strcpy(player->full_name, namebuf,
-								  sizeof(player->full_name));
+                    if(arg_force_name)
+                        msg("You are not allowed to change your name!");
+                    else {
+                        char namebuf[32] = "";
 
+                        /* Set player name */
+                        if (get_character_name(namebuf, sizeof namebuf))
+                            my_strcpy(player->full_name, namebuf,
+                                      sizeof(player->full_name));
+                    }
 					break;
+                    
 				}
 
 				case 'f': {
 					char buf[1024];
 					char fname[80];
 
-					/* Get the filesystem-safe name and append .txt */
-					player_safe_name(fname, sizeof(fname), player->full_name, false);
-					my_strcat(fname, ".txt", sizeof(fname));
+                    /* Get the filesystem-safe name and append .txt */
```

Comment:
As far as I can tell, this change is just changing the indentation of this code - please could you change it back to using tabs?